RUFAS/routines/animal/types/preg_check_config.py

```diff
@@ -0,0 +1,22 @@
+from typing import TypedDict
+
+
+class PregCheckConfig(TypedDict, total=False):
+    """
+    List of expected keys for preg check configuration dictionary,
+    used in daily pregnancy check routine for HeiferII and Cow classes.
+
+    Each dictionary contains:
+        - 'day': The number of days from the AI day when the pregnancy check should occur.
```

Comment:
```suggestion
        - 'day': The number of days from the day of artificial insemination when the pregnancy check should occur.
```
I'm pretty sure this is what AI means in this context, I think it would be better to make it a little more verbose.